src/main/java/org/magicdgs/thaplv/haplotypes/model/HaplotypeModel.java

```diff
@@ -75,4 +76,25 @@
         }
     }
 
+    /**
+     * {@link VariantHaplotypeConverter} for don't check genotypes
+     */
+    private static final class DontCheckHaplotypeConverter implements VariantHaplotypeConverter {
```

Comment:
Open a ticket for checking every _x_ variants when no check: faster than converting but required for thrown an error